CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [15.2.0]
+### Fixed
+- Fix bug that allowed for multiple fetch requests instantiate from getGasFeeEstimatesAndStartPolling ([#586](https://github.com/MetaMask/controllers/pull/586))
+### Changed
+- Enable default caching for token-service ([#594](https://github.com/MetaMask/controllers/pull/594))
+- Remove synctoken ([#590](https://github.com/MetaMask/controllers/pull/590))
```

Comment:
@Gudahtt would this be considered a breaking change even though we never consumed it (atleast within Metamask - I think)?

Reply:
Yes I think so. We've treated "breaking" changes to unused APIs as breaking in the past as well. We're not the only people that use this library after all.